AsyncDisplayKit/ASMapNode.mm

```diff
@@ -41,30 +42,40 @@ - (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate
   _centerCoordinateOfMap = kCLLocationCoordinate2DInvalid;
 
   _options = [[MKMapSnapshotOptions alloc] init];
-  _options.region = MKCoordinateRegionMakeWithDistance(coordinate, 1000, 1000);;
+  _options.region = region;
   
   return self;
 }
 
-- (void)setAnnotations:(NSArray *)annotations
+- (void)didLoad
 {
-  ASDN::MutexLocker l(_propertyLock);
-  _annotations = [annotations copy];
-  if (annotations.count != _annotations.count) {
-    // Redraw
-    [self setNeedsDisplay];
-  }
+    [super didLoad];
+    if (self.isLiveMap && !_mapNode) {
+        self.userInteractionEnabled = YES;
+        [self addLiveMap];
+    }
 }
 
-- (void)setUpSnapshotter
+- (void)fetchData
 {
-  if (!_snapshotter) {
-    ASDisplayNodeAssert(!CGSizeEqualToSize(CGSizeZero, self.calculatedSize), @"self.calculatedSize can not be zero. Make sure that you are setting a preferredFrameSize or wrapping ASMapNode in a ASRatioLayoutSpec or similar.");
-      _options.size = self.calculatedSize;
-      _snapshotter = [[MKMapSnapshotter alloc] initWithOptions:_options];
-  }
+    [super fetchData];
+    if (_liveMap && !_mapNode) {
+        [self addLiveMap];
```

Comment:
Make sure all indentation throughout the file is 2 spaces, to match ASDK code style.  You can set this in your editor and then use the "re-indent" function if you select all in the file.